### ISSUE TITLE
Model weights and gradients in TrainKeras

### ISSUE THREAD

**Issue by brollb** [deepforge-dev/deepforge#1988]
It would be nice to visualize the model weights and gradients in TrainKeras as this can be useful in detecting things like vanishing/exploding gradients.

**Comment by brollb** [comment]
Should these be visualized on the training architecture? 
The approach used with pipelines/executions could be useful here, too. The neural network architectures could be augmented with "execution" information such as model weights which can then be visualized... Maybe this would be best to add as metadata to trained Keras models which can be used to view model checkpoints.
